src/qt/editor/helpwidget.cpp

```diff
@@ -204,6 +207,7 @@ void link(T& factory, std::string_view id, Document::DocumentHandle& handle) {
     if (auto* fo = factory->getFactoryObject(id)) {
         auto& typeName = fo->getTypeName();
         std::string name{typeName.substr(0, typeName.find_first_of('<'))};
+        replaceInString(name, "class ", "");
```

Comment:
not sure if this is the right place...

Reply:
there might also be structs... not use if it should be in the base or not...

Reply:
Would have used `util::demangle()` which, according to the docs, will remove `class ` but doesn't.

https://github.com/inviwo/inviwo/blob/bbef904fa5898e276ca5b13ba0f6c2413c3779bc/include/inviwo/core/util/demangle.h#L36-L41

Reply:
I think demangle should probably remove both class and struct on msvc